src/statistics/PearsonCorrelaion.jl

```diff
@@ -0,0 +1,27 @@
+"""
+    Pearson Correlation
+
+Find the pearson correlation between two variables.
+
+# Example:
+
+julia> PearsonCorrelation([12,11,16,17,19,21],[11,51,66,72,12,15])
+-0.2092706263573845
+
+Contribution by: [Aru Bhardwaj](https://github.com/arubhardwaj)
+
+
+"""
+
+
+
+function PearsonCorrelation(x, y)
+    mean_x = sum(x)/length(x)
+    mean_y = sum(y)/length(y)
+    XY = (x.-mean_x).*(y.-mean_y)
+    XXs = sum((x.-mean_x).*(x.-mean_x))
+    YYs = sum((y.-mean_y).*(y.-mean_y))
+    return(sum(XY)/(sqrt(XXs.*YYs)))
+end
+
+
```

Comment:
```suggestion
```

Reply:
what suggestions it is?

Reply:
This is to remove two unused lines at the end. 🙂

Reply:
well, how it will affect the code? Would it impact anything? Just curious

Reply:
> well, how it will affect the code? Would it impact anything? Just curious

No, it won't affect anything, but it's just for better-looking. 🙂

Reply:
well